src/build123d/topology.py

```diff
@@ -7000,6 +6982,26 @@ def make_loft(
         """
         return cls(_make_loft(objs, False, ruled))
 
+    def thicken(self, depth: float) -> Solid:
+        """Thicken Shell
+
+        Create a solid from a shell by thickening along the normals.
+
+        Args:
+            depth (float): Amount to thicken face(s), can be positive or negative.
+            normal_override (Vector, optional): The normal_override vector can be used to
```

Comment:
The `Shell` version of `thicken` doesn't have the `normal_override` parameter so it needs to be removed from the docstring.